yarn-project/circuit-types/src/logs/l1_note_payload/l1_note_payload.ts

```diff
@@ -57,27 +56,21 @@ export class L1NotePayload {
   /**
    * Encrypt the L1NotePayload object using the owner's public key and the ephemeral private key.
    * @param ownerPubKey - Public key of the owner of the L1NotePayload object.
-   * @param curve - The curve instance to use.
    * @returns The encrypted L1NotePayload object.
    */
-  public toEncryptedBuffer(ownerPubKey: PublicKey, curve: Grumpkin): Buffer {
+  public toEncryptedBuffer(ownerPubKey: PublicKey): Buffer {
     const ephPrivKey: GrumpkinPrivateKey = GrumpkinScalar.random();
-    return encryptBuffer(this.toBuffer(), ownerPubKey, ephPrivKey, curve);
+    return encryptBuffer(this.toBuffer(), ownerPubKey, ephPrivKey);
```

Comment:
Like the others we will likely need to alter this as the ephPrivKey or `ephSk` will be used multiple places so cannot just throw away early.